src/Editor.vue

```diff
@@ -61,6 +61,7 @@ const highlight = (lang, jsxInExamples) => {
 
 export default {
   name: "VueLiveEditor",
+  inheritAttrs: false,
```

Comment:
Just curious. Why was this one necessary?

Reply:
Otherwise the @change defined in VueLive.vue is inherited by PrismEditor component in Editor.vue. Then, when the editor value changes, it trigger the event twice; once by PrismEditor textarea (payload: Event, undesired) and once by the Editor (payload: code string, desired).

Reply:
Oh !! good call.